### DISCUSSION TITLE
128MB 的 ROM，是不是刷不了ImmortalWrt？

### DISCUSSION THREAD

**Discussion by Rangtian** [immortalwrt/immortalwrt#1627] [Q&A]
Redmi AC2100，128MB 的 ROM，在 [ImmortalWrt Firmware Selector](https://firmware-selector.immortalwrt.org/) 下的固件，通过 Breed 刷入。

试了好几个版本，都能成功刷入 kernel 包。可登录 kernel 包后，再利用 ImmortalWrt 本身的系统升级刷 sysupgrade 包，无一例外地失败了。192.168.1.1 无法访问，手机里也找不到 ImmortalWrt 相关的无线信号。

刷 kernel 包时路由器的灯好歹还变成橙色闪几下，刷 sysupgrade 路由器的灯一直是蓝色，也不闪。最后实在没法子，又刷回了 Padavan。

请教各位，是不是 ROM 太小刷不了？

**Comment by lambdaplus** [comment]
应该是你操作出错了，或者固件选错了。

**Comment by ponycloud235** [comment]
好像是因为空间不足，我刷了几次都没有成功过

**Comment by lonelygo** [comment]
能刷成功，但是 wrt 启动后太吃内存了，128 基本上给系统用完了，别的东西就没法弄了，比如🐱

**Reply by ponycloud235** [reply]
能刷。 也能用机场。 就是偶尔会卡。

**Reply by lonelygo** [reply]
> 能刷。 也能用机场。 就是偶尔会卡。
兄弟用的nikki还是shellcrash或者其他的？
我不知道是不是我配置文件太复杂了，起来就是50M朝上走，基本上服务起来就oom，路由器就重启了。
毛子固件上也没啥好用的🐱，现在Redmi AC2100只能吃灰了。

**Reply by ponycloud235** [reply]
那我倒没有重启，是用Nikki。 这个机器确实性能太差，拿来做玩具。 我主的还是用软路由 x64的。  可以刷个老毛子，做ap用。
